### PR TITLE
Docs: Update README to build using Makefile

### PR DESCRIPTION
**Pull request by carlosperate**
Makefile now created the microbitversion.h file needed for yt build to complete successfully.

### CODE REVIEW

**Comment by carlosperate** [comment]
Alternatively we could also create a dummy header file to be overwritten by the makeversionhdr.py script.

**Comment by dpgeorge** [comment]
> Alternatively we could also create a dummy header file to be overwritten by the makeversionhdr.py script.

I don't think that would work well because git will always see the dummy file as modified after a build.  I think we've accepted by now that we need to use `make` to build the firmware fully.

**Comment by carlosperate** [comment]
>I don't think that would work well because git will always see the dummy file as modified after a build. I think we've accepted by now that we need to use make to build the firmware fully.

That's a very good point, we'll have to use make.
Does the PR need any changes?

File: README.md [hunk 1]
```diff
@@ -43,16 +43,16 @@ as a part of the process.
...
-repository.
+The resulting firmware.hex file to flash onto the device can be
+found in the build/ directory from the root of the repository.
 
-There is a Makefile provided that does some extra preprocessing of the source,
-which is needed only if you add new interned strings to qstrdefsport.h.  The
-Makefile also puts the resulting firmware at build/firmware.hex, and includes
+The Makefile provided does some extra preprocessing of the source,
+it adds version information to the UICR region, and process new interned
+strings to qstrdefsport.h.
```
Comment (dpgeorge) [inline README.md]: The last part of the sentence here about qstrdefsport.h is no longer true, so can be removed.

Comment (carlosperate) [reply README.md]: 👍 
Commit updated.

**Comment by dpgeorge** [comment]
Thanks for updating, merged in 55e906c4abb629933085fa06693897435823ce62

**Pull request closed without merging**